hoomd/box.py

```diff
@@ -80,11 +175,11 @@ class Box:
 
     .. rubric:: Two dimensional systems
 
-    2D simulations in HOOMD use boxes with ``Lz == 0``. 2D boxes ignore ``xz``
-    and ``yz``. If a new `Box` is assigned to a system with different
-    dimensionality, a warning will be shown.
+    Set ``Lz == 0`` to make the box 2D. 2D boxes ignore ``xz`` and ``yz``.
+    Changing the box dimensionality from 2D to 3D (or from 3D to 2D) during
+    a simulation will result in undefined behavior.
 
-    In 2D boxes, *volume* is in units of area.
+    In 2D boxes, *volume* is in units of :math:`[\\mathrm{length}]^2`.
 
     .. rubric:: Factory Methods
```

Comment:
the examples in this section are rendering without line breaks in between them.

Reply:
I have fixed this multiple times, I don't know what keeps undoing the fix. Will try again.

Reply:
Now I know: This uses the "Section:" Google syntax which should not start with a newline. However, restructured text bulleted lists must have a blank line above them. I reformatted this as a rubric.